docs/notebooks/markov-models.ipynb

```diff
@@ -137,7 +137,7 @@
     "The symbol etymology, which usually gets swept under the rug in mathematically-oriented papers, are as follows: \n",
```

Comment:
<p>Shouldn't it be <code>mu = k*y_{t-1} + mu_St</code>  (instead of <code>mu_t</code>  currently)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ericmjl/bayesian-analysis-recipes/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>Yeah, good point. I'm sloppy with symbols. It's why I was doing badly in math. Give me Python, though.... :)</p>